### ISSUE TITLE
getCheermotes.parseMessage() throws an error

### ISSUE THREAD

**Issue by that1estonian** [twurple/twurple#294]
### Code

```javascript
const getCheermotes = await twitchApi.bits.getCheermotes('423142326')
const cheermotes = getCheermotes.parseMessage(message)
```

### Expected behavior

I expected to get the cheermotes from the message. Worked on 4.6.7

### Actual Behavior

```
C:\Users\***\***\node_modules\@twurple\api\lib\api\helix\bits\HelixCheermoteList.js:26
        const { background, state, scale } = format;
                ^

TypeError: Cannot destructure property 'background' of 'format' as it is undefined.
```

### Environment

- Version: 5.0.1
- Node version: v16.9.1
- Operating system: Windows 11

**Comment by d-fischer** [comment]
I just published 5.0.2 to fix this.

This also means that `parseMessage` got a *required* additional parameter that defines the format, since its defaults were removed from the underlying method, but the implemented interface still didn't include it. (It does now.)

This is *technically* a breaking change, but I think it's fine for a patch version since the method was not working at all before.

The change was also added to the migration guide.
